tests/e2e-leg-6/revivedb-1-node-restore/vdb-to-create/base/setup-vdb.yaml

```diff
@@ -0,0 +1,35 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
```

Comment:
Lets use the v1 API. A few things, you will need to set:
- `vertica.com/include-uid-in-path` annotation as we removed `communal.includeUIDInPath` parm
- rename `superuserPasswordSecret` to `passwordSecret`
- `vertica.com/k-safety` annotation as we removed the `kSafety` parm

Reply:
will do